pkg/server/login/login.go

```diff
@@ -128,6 +130,8 @@ func (l *Login) handleLoginForm(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
+	form.Locale = getLocale(getPrefferedLang(req.Header.Get("Accept-Language")))
```

Comment:
What if I want to specify the language myself, manually? That's actually very often the case for myself, I default my system to English language, but prefer websites on the Czech web to be in Czech, which oftentimes I have to do manually on Czech pages since the autodetection is wrong.

I would expect some kind of this + a language button choice of which is stored in a cookie.

Reply:
> What if I want to specify the language myself, manually

if we have locale for that lang it will be used, if not we default to English

>I would expect some kind of this + a language button choice of which is stored in a cookie.

This story only contains reading the lang preferences from the header and parsing the go template with appropriate locale.

Reply:
@stlaz We have this for console itself. The problem is that it's stored as a user preference, which is tied to a specific user. Here you haven't logged in yet, so we can't load your user preferences.

We could also store the last selected language in localStorage and pass it as a query parameter to the login page, however. Then the query parameter would override `Accept-Language` if present. @jhadvig Do you might also checking a `lng` query parameter (to match what we support in console) first, then falling back to `Accept-Language`?

Note that "preferred" is misspelled here :)